File: src/level1/SubakSubakSu.go
func solution(n int) string {
    a := "수"
    b := "박"
    c := ""
    for i := 1; i <=n; i++{
        if i % 2 == 0 {
           c += b 
        }else{
            c += a
        }
    }
    return c
}

File: src/level1/SubakSubakSu.js
function solution(n) {
    var answer = '';
    var su ='수';
    var bak = '박';
    for(var i = 1; i <= n; i++){
        if(i % 2 == 0){
            answer += bak;
        }else{
            answer += su;
        }
    }
    return answer;
}
